lib/roger/resolver.rb

```diff
@@ -176,7 +176,7 @@ def sanitize_name(name, prefer = nil)
       path = name.to_s
 
       # If it's a directory append "index"
-      path = File.join(path, "index") if File.directory?(name)
+      path = File.join(path, "index") if File.directory?(name) && Dir.glob(name + ".*").none?
```

Comment:
Hmmm. Ok things are getting clearer now. This code is designed for this:

```
- html
  - bla/
    - index.html.erb
  - bla.html.erb
```

But in the case there is no `html/bla/index.html.erb` you want it to find `html/bla.html.erb`. Or do you always want `html/bla.html.erb` to have preference over `html/bla/index.html.erb`?